src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Email } from '../Email/Email';
+import { Section } from '../Section/Section';
+import { Column } from '../Column/Column';
+import { Button } from './Button';
+
+export default {
+  component: Button,
+} as ComponentMeta<typeof Button>;
+
+//“template” of how args map to rendering
+const Template: ComponentStory<typeof Button> = (args) => (
+  <Email>
+    <Section>
+      <Column>
+        <Button {...args} />
+      </Column>
+    </Section>
+  </Email>
+);
+
+export const Default = Template.bind({});
+export const Primary = Template.bind({});
+export const Secondary = Template.bind({});
```

Comment:
Let's keep the `Template.bind` and `args` together.

also, let's go like this.
```jsx
const defaultArgs = {
  children: 'Primary Button',
  href: 'https://github.com/leopardslab/react-email',
  variant: 'primary',
  classes: {
    primary: {
      backgroundColor: 'green',
      color: 'white',
    },
  },
}

export const Default = Template.bind({});
Default.args = defaultArgs;

export const Primary = Template.bind({});
Primary.args = {
 ...defaultArgs,
variant: 'primary'
};
```